### ISSUE TITLE
can't find element by props in 1.9.0

### ISSUE THREAD

**Issue by mgrybyk** [baruchvlz/resq#64]
**Steps to reproduce**
_in browser console_
`resq.resq$$('*', document.getElementById('root')).byProps({name: '5'})`

**Actual Result**
returned 44 elements

**Expected Result**
one element

**WebdriverIO steps to reproduce**
```js
browser.url('https://ahfarmer.github.io/calculator/')
const el = browser.react$('*', { props: { name: '5' } })
console.log(el.getTagName(), el.getText())
```

**Actual Result**
```
div 0
AC
+/-
%
÷
7
8
9
x
4
5
6
-
1
2
3
+
0
.
=
```

**Expected Result**
`div 5`

**Comment by mgrybyk** [comment]
@baruchvlz will you be able to take a look?

**Comment by baruchvlz** [comment]
I checked with 1.8.0, seems to be working fine. The changes done to `verifyIfObjectsMatch` function is what's screwing this up, will try to have a fix soon.

**Comment by baruchvlz** [comment]
Should be fixed in `v1.9.1`
